taxes/get-tax-controller.js

```diff
@@ -0,0 +1,25 @@
+const Calculator = require('./calculator')
+const TaxRegulator = require('./tax-regulators')
+
+class GetTaxController {
+    process(ctx) {
+        const body = ctx.request.body;
+        if (!body.state || !body.productPrice) {
+            return ctx.body = { error: "Required parameters not present" };
+        }
+
+        try {
+            const calculatorResult = new Calculator(body.productPrice, body.state).calculate();
```

Comment:
if the calculator is an object dependency, why do you need to create it inside the method